applications/working_with_helm_charts/odc-working-with-helm-releases.adoc

```diff
@@ -6,13 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-
 You can use the *Developer* perspective in the web console to upgrade, rollback, or uninstall a Helm release.
```

Comment:
```suggestion
You can use the *Developer* perspective in the web console to update, rollback, or uninstall a Helm release.
```
Per the contributor guidelines https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update